plugins/module_utils/table.py

```diff
@@ -15,8 +15,9 @@ def _path(table, *subpaths):
 
 
 def _query(original=None):
-    # Flatten the response (skip embedded links to resources)
-    return dict(original or {}, sysparm_exclude_reference_link="true")
+    original = original or dict()
+    original["sysparm_exclude_reference_link"] = original.setdefault("sysparm_exclude_reference_link", "true")
```

Comment:
```suggestion
    original.setdefault("sysparm_exclude_reference_link", "true")
```

Reply:
While correct, this assignment is unnecessary.

Reply:
Resolved with https://github.com/ansible-collections/servicenow.itsm/pull/182/commits/1e80f599254e4e166f41c84fd9f125120224fd47.